src/components/PrivacyPolicy.tsx

```diff
@@ -0,0 +1,31 @@
+import ExternalLink from './shared/ExternalLink'
+
+export default function PrivacyPolicy() {
+  return (
+    <div style={{ padding: '40px', maxWidth: '1080px', margin: '0 auto' }}>
+      <h1>Privacy Policy</h1>
+      <p>
+        We want to process as little personal information as possible when you
```

Comment:
quick q: Do we wanna translate this content?

Reply:
Because this is such a big piece of text and likely to be changed soon, I think it should be left unmarked

Reply:
Gotta be careful translating stuff like this too. Might need a bit more of a stringent translation review process.